rotkehlchen/api/rest.py

```diff
@@ -2523,6 +2524,18 @@ def add_rpc_node(self, node: WeightedNode) -> Response:
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
     def update_rpc_node(self, node: WeightedNode) -> Response:
+        # get the current rpc endpoint so we can identify the node and remove it
```

Comment:
This function absolutely needs a docstring and perhaps a rename. It seems to do more than update the rpc node. It also connects. So it's more like a `update_and_connect_rpc_node` ?